### PR TITLE
Add communication and ui elements

### PR DESCRIPTION
**Pull request by bidoubiwa**
## In this PR

- [x]  Base UI elements closes: #10 
    - [x] Host and Keys (stored in db)
    - [x] Index name input
    - [x] Documents input json 
- [x]  Back end communication with MeiliSearch closes: #9 

closes: #11

### TODO
- [x] remove test button top right
- [x] remove comments

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,14 +7,14 @@
     "style:fix": "eslint --ext .js . --fix",
     "playground:build": "yarn --cwd ./playground && yarn --cwd ./playground build",
     "playground:dev": "yarn --cwd ./playground && yarn --cwd ./playground develop"
-
   },
   "strapi": {
     "name": "meilisearch",
     "icon": "plug",
-    "description": "Description of meilisearch plugin."
+    "description": "Descript on of meilisearch plugin."
```
Comment (bidoubiwa) [inline package.json:14]:
```suggestion
    "description": "Strapi plugin for MeiliSearch"
```

**Review by curquiza — COMMENTED** [review]
I've noticed some naming inconsistencies:

- `autoFillMsCredentials`
- `addMeilisearchCredentials`
- `getMeiliSearchCredentials`
=> `Ms` vs `Meilisearch` vs `MeiliSearch`

Otherwise, this PR is ok to me! 🚀

**Review by curquiza — COMMENTED** [review]
I see you added a commit so you might want to make consistency between namings, but I see there are still `addMeilisearchCredentials` vs `addMeiliSearchCredentials`.

If it's expected, ignore my comment and merge 🚀

**Comment by bidoubiwa** [comment]
Oh my bad. I did not see the capital S

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**